test-cases/longevity/longevity-ics-nemesis-alter-table-1TB-7days.yaml

```diff
@@ -0,0 +1,37 @@
+test_duration: 10900
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=1100200300 -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native  -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..1100200300",
```

Comment:
Change to -col 'size=FIXED(200) n=FIXED(5)' so you have 1KB rows with 5 columns.
change 1100200300 to 500,000,000

Reply:
done